bindings/python/pydeck/pydeck/bindings/json_tools.py

```diff
@@ -52,6 +52,9 @@ def default_serialize(o, remap_function=lower_camel_case_keys):
     for ignore_attr in IGNORE_KEYS:
         if attrs.get(ignore_attr):
             del attrs[ignore_attr]
+    if attrs.get('type'):
```

Comment:
Is this some backwards compatibility? Maybe add a comment to explain? Possible to just warn instead rather than keeping complex logic?